config/foreman.migrations/20220406150414_foreman_google.rb

```diff
@@ -0,0 +1,5 @@
+answers['foreman::plugin::google'] ||= false
+
+if answers.key?('foreman::compute::gce')
+  answers.delete('foreman::compute::gce')
```

Comment:
Wouldn't it make sense to assume that if previously someone used gce, they now want the google plugin?

Reply:
Yop, make sense, updated.

However `rspec ./spec/migration_spec.rb` tests are still failing on mismatch with answers, not sure why